File: ejercicio_26/ejercicio_25.java
package programacion_practica.ejercicio_25;
import java.util.Scanner;
public class ejercicio_25 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        String cadena;
        System.out.print("Ingresa la cadena: ");
        cadena = entrada.next();
        String complemento = ""; 
        for (int i = 0; i < cadena.length(); i++) {
            char simbolo = cadena.charAt(i);
            if (simbolo == 'A') {
                complemento += 'T';
            } else if (simbolo == 'T') {
                complemento += 'A';
            } else if (simbolo == 'C') {
                complemento += 'G';
            } else if (simbolo == 'G') {
                complemento += 'C';
            }
        }
        System.out.println("Cadena complementaria: "+complemento);
        entrada.close();
    }   
}

File: ejercicio_26/ejercicio_25.py
cadena = input("Ingresa la cadena: ")
complemento = ""
for simbolo in cadena:
    if simbolo == 'A':
        complemento += 'T'
    elif simbolo == 'T':
        complemento += 'A'
    elif simbolo == 'C':
        complemento += 'G'
    elif simbolo == 'G':
        complemento += 'C'
print("Cadena complementaria:", complemento)
